docs/en/guides/How-to-build.md

```diff
@@ -51,18 +51,7 @@ or
 ### Build docker images
 We can build docker images of `backend` and `ui` with `Makefile` located in root folder.
 
-- Build all docker images
-> make docker.all
-
-- Build oap server docker image
-> make docker.oap
-
-- Build ui docker image
-> make docker.ui
-
-`HUB` and `TAG` variables ares used to setup `REPOSITORY` and `TAG` of a docker image. To get
-a oap image with name `bar/oap:foo`, run the following command
-> HUB=bar TAG=foo make docker.oap
+Refer to [Build docker image](https://github.com/apache/skywalking/tree/master/docker) for more details.
```

Comment:
I think we should add refer to `makefile` doc?

Reply:
The usage of `makefile` is written in `docker/README.md`.  When the user navigates to this folder, they can view documents and scripts at the same time, I think it's not a bad experience for them.

Reply:
I know the doc is there. But at here, the doc said, 

> We can build docker images of `backend` and `ui` with `Makefile` located in root folder.

There is no tip about reading the doc there.